src/frontend/src/common/components/library/data_subview/components/DownloadModal/index.tsx

```diff
@@ -173,43 +183,69 @@ const DownloadModal = ({
                   anchorEl,
                 }}
               >
-                <StyledSpan style={getBackgroundFastaColor()}>
-                  <CheckBoxWrapper>
-                    <CheckBoxInfo>
-                      {/* @ts-expect-error need to update checkbox state after upgrading sds */}
-                      <StyledCheckbox
-                        onChange={handleFastaClick}
-                        disabled={isFastaDisabled}
-                      />
-                    </CheckBoxInfo>
-                    <CheckBoxInfo>
-                      <DownloadType style={getBackgroundFastaColor()}>
-                        Consensus Genome{" "}
-                      </DownloadType>{" "}
-                      <span ref={tooltipRef}>(.fasta)</span>
-                      <DownloadTypeInfo>
-                        Download multiple consensus genomes in a single,
-                        concatenated file
-                      </DownloadTypeInfo>
-                    </CheckBoxInfo>
-                  </CheckBoxWrapper>
-                </StyledSpan>
+                <StyledFileTypeItem
+                  isDisabled={isFastaDisabled}
+                  isSelected={isFastaSelected}
+                >
+                  <CheckBoxInfo>
+                    <StyledCheckbox
+                      disabled={isFastaDisabled}
+                      id="download-fasta-checkbox"
+                      onChange={handleFastaClick}
+                      stage={isFastaSelected ? "checked" : "unchecked"}
+                    />
+                  </CheckBoxInfo>
+                  <CheckboxLabel htmlFor="download-fasta-checkbox">
```

Comment:
Accessibility: Checkboxes need to have labels.  In our case, we had a visible label, but the screen reader didn't know to connect the two.  Now we have a semantic `label` element and the `htmlFor` field connects it to the checkbox element above.  Now the screen reader announces the label 🎉